### ISSUE TITLE
Decoding Error

### ISSUE THREAD

**Issue by CpHarding** [raimon49/pip-licenses#37]
Same issue as:
*  Decoding Error when reading license file. #35
but setting LANG in windows appears to do nothing - meaning you can't run pip-licenses!

Can we get a command line argument to override the default encoding please?

**Comment by raimon49** [comment]
@CpHarding Thanks for reporting the issue.

I want to know your specific Python environment to resolve this issue. It is the Python version and architecture.

For example, what does the Python interpreter start up display?

```sh
C:\work>py
Python 3.7.1 (v3.7.1:260ec2c36a, Oct 20 2018, 14:57:15) [MSC v.1915 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>>
```

**Comment by erinpentecost** [comment]
I'm getting a similar issue for [this license file](https://github.com/datadriventests/ddt/blob/master/LICENSE.md).
```sh
>py
Python 3.7.3 (v3.7.3:ef4ec6ed12, Mar 25 2019, 21:26:53) [MSC v.1916 32 bit (Intel)] on win32
Type "help", "copyright", "credits" or "license" for more information.
```

**Comment by raimon49** [comment]
@erinpentecost Thanks for the report. I reproduced it in my Windows environment.

This issue is resolved in the next release 1.15.2.
